ajax/libs/country-region-dropdown-menu/package.json

```diff
@@ -20,14 +20,14 @@
     "target": "git://github.com/geodatasource/country-region-dropdown-menu.git",
     "fileMap": [
       {
-        "basePath": "dist",
+        "basePath": "assets/js",
```

Comment:
As there are many files there, maybe we need to add the whole `assets` ?

Reply:
Sure, will update about it. Is there any way to add the [language](https://github.com/geodatasource/country-region-dropdown-menu/tree/master/languages) folder too?

Reply:
Yes, with another filemap block.